src/redux.js

```diff
@@ -92,20 +90,12 @@ export function setSelectedTrainer(selectedTrainer) {
   return { type: SET_SELECTED_TRAINER, selectedTrainer };
 }
 
-export function setKValue(kValue) {
-  return { type: SET_K_VALUE, kValue };
-}
```

Comment:
I think we can also remove line 36 `const SET_K_VALUE = "SET_K_VALUE"; and then the kValue in initial state and also where we reference that state in the algorithm.  I'd even prefer if deleting `setKValue` and its tendrils lived in its own PR.

Reply:
Prepped a standalone PR in https://github.com/code-dot-org/ml-playground/pull/146 which can be merged first.